### PR TITLE
New title and description for zenodo

### PR DESCRIPTION
**Pull request by nabobalis**

### CODE REVIEW

File: .zenodo.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-    "description": "The community-developed, free and open-source solar data analysis environment for Python.",
+    "description": "sunpy is a software library that provides fundamental tools for accessing, loading and interacting with solar physics data in Python",
     "license": "BSD-2-Clause",
-    "title": "SunPy",
+    "title": "SunPy Core Package: All your Heliophysics needs",
```
Comment (nabobalis) [inline .zenodo.json:4]:
```suggestion
    "title": "SunPy Core Package: For all your solar physics needs",
```

File: .zenodo.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
-    "description": "The community-developed, free and open-source solar data analysis environment for Python.",
+    "description": "sunpy is a software library that provides fundamental tools for accessing, loading and interacting with solar physics data in Python",
     "license": "BSD-2-Clause",
-    "title": "SunPy",
+    "title": "SunPy Core Package: For all your solar physics needs",
```
Comment (nabobalis) [inline .zenodo.json:4]:
```suggestion
    "title": "SunPy Core Package: sunpy - For all your solar physics needs",
```

Comment (wtbarnes) [reply .zenodo.json:4]: ....no?

Comment (wtbarnes) [inline .zenodo.json:4]:
```suggestion
    "title": "SunPy",
```

**Comment by wtbarnes** [comment]
What is the point of this PR?

File: .zenodo.json (hunk 2 above)
Comment (nabobalis) [reply .zenodo.json:4]: But this isn't SunPy.

**Comment by nabobalis** [comment]
To correct the incorrect title of the zendo entry

File: .zenodo.json (hunk 2 above)
Comment (nabobalis) [inline .zenodo.json:4]:
```suggestion
    "title": "SunPy Core Package: sunpy",
```

Comment (wtbarnes) [reply .zenodo.json]: Why not just `sunpy`?

Comment (nabobalis) [reply .zenodo.json]: Works for me.

File: .zenodo.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
-    "description": "The community-developed, free and open-source solar data analysis environment for Python.",
+    "description": "sunpy is a software library that provides fundamental tools for accessing, loading and interacting with solar physics data in Python",
     "license": "BSD-2-Clause",
-    "title": "SunPy",
+    "title": "SunPy Core Package: sunpy",
```
Comment (Cadair) [inline .zenodo.json:4]: I don't think this is going to work for anyone other than the 4 of us who get picky about `sunpy` vs `SunPy`. I think this should be `The SunPy Core Package` or `sunpy: A Core Package for Solar Physics` or something

Comment (nabobalis) [inline .zenodo.json:4]:
```suggestion
    "title": "sunpy: A Core Package for Solar Physics",
```

**Review by Cadair — APPROVED** [review]

**Pull request merged**